static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -316,16 +316,34 @@ function DBQueryInjectionVulnerabilityEvidence({
   organization,
   location,
   projectSlug,
+  offendingSpans,
 }: SpanEvidenceKeyValueListProps) {
   const evidenceData = event?.occurrence?.evidenceData ?? {};
+  const formattedVulnerableParameters = evidenceData.vulnerableParameters?.map(
+    (param: string[]) => {
+      if (typeof param[1] === 'object') {
+        return `${param[0]}: ${JSON.stringify(param[1])}`;
+      }
```

Comment:
wondering if this needs to check the performance issue type or anything

Reply:
this is already in `DBQueryInjectionVulnerabilityEvidence` so it'll only happen for this group type